cou/steps/__init__.py

```diff
@@ -50,10 +50,10 @@ def compare_step_coroutines(coro1: Optional[Coroutine], coro2: Optional[Coroutin
     )
 
 
-class UpgradeStep:
+class BaseStep:
```

Comment:
We are missing `UpgradePlan`,  which should by similar to `ApplicationUpgradeStep`.